prov/verbs/src/fi_verbs.h

```diff
@@ -580,11 +574,22 @@ struct vrb_ep {
 		};
 	};
 
-	size_t				inject_limit;
-
+	struct {
+		size_t			inject_size;
+		size_t                  tx_size;
+		size_t                  tx_iov_limit;
+		size_t                  rx_size;
+		size_t                  rx_iov_limit;
+		uint32_t                protocol;
+		uint32_t                addr_format;
+		size_t                  src_addrlen;
+		size_t                  dest_addrlen;
+		void                    *src_addr;
+		void                    *dest_addr;
+		void                    *handle;
+	} info_attr;
 	struct vrb_eq			*eq;
 	struct vrb_srq_ep		*srq_ep;
-	struct fi_info			*info;
```

Comment:
Once the underlying QP has been setup, is there any reason to keep this?  Can we just free the info structure in that case as an alternative change?

Reply:
I don't think it can be done yet. There are some defines and checks that depend on variables like inject_size. Most of the rest might be able to go away though.

Reply:
Nevermind. Steve basically said the same thing below.